AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_TAN_DOWA;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on tracking with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of the Tan Dowa. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_TANIT;It is early morning and you are at the shrine dedicated to Tanit, deep in prayers. You hear soft footsteps behind you - who could it be at this hour? [From.GetTitledFirstName], as it turns out. You exchange some pleasantries and soon you find yourself in a full-fledged conversation. You have so much in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_ANTI;Today [From.GetTitledFirstName] has invited you to watch a wrestling match between two local champions, after the first round of the competition you have had time to talk and soon discovered that [From.GetSheHe] is also a devotee of �nti, the great giant fighter.;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_GURZIL;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on archery with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of Gurzil. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_MAGEC;It is said that the nobles are pale because they spend all their time indoors, but not you. As a devotee of Magec the sun you love walking through the fields enjoying its light. On one of your walks you meet [From.GetTitledFirstName] who is also dedicating [From.GetHerHis] day to Magec. It is always good to find another devotee.;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_SET;When you needed an accomplice in a small scheme, [From.GetTitledFirstName] proved to be an unexpected ally. Think what mutual benefits can do to unite two souls! While it was pure business at first, it turns out you have plenty in common, such as your reverence of Set. Now you spend more time conversing than plotting...;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_SAMARI;You are dancing at the secret rite held in the Samari's honor as your eyes find [From.GetTitledFirstName]'s across the bonfire. For a moment, it is as if the whole world turns silent. The drumming is a distant thunder, yet the rhythmic vibrations run through your body, to your very core...;;;;;;;;;;;;;x
+EVTOPTA_107_SAMARI;Tonight, I shall bed [From.GetFirstName] in the name of the Samari.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7510_SAMARI;Lately, you have striven to act in the spirit of Tanit by treating your [From.GetHusbandWife] [From.GetTitledFirstName] with respect and warmth. Slowly it has changed the dynamic of your relationship, and those who are closest to you have noticed it as well.;;;;;;;;;;;;;x
```

Comment:
Looks like you mistakenly put that key here instead, when this text is about Tanit.